src/actions.rs

```diff
@@ -0,0 +1,886 @@
+//! Actions functionality for WebDriver.
+use crate::elements::Element;
+use std::fmt::Debug;
+use std::time::Duration;
+use webdriver::actions as WDActions;
+
+/// An action not associated with an input device (i.e. pause).
+#[derive(Debug, Clone)]
+pub enum NullAction {
+    /// Pause for the specified duration.
+    Pause {
+        /// The pause duration, given in milliseconds.
+        duration: u64,
+    },
+}
+
+impl From<NullAction> for WDActions::NullActionItem {
```

Comment:
Unfortunately all of these `From` impls must be replaced with private inherent methods instead. See my comment elsewhere for why.